ui/src/app/base/reducers/machine/machine.js

```diff
@@ -35,6 +35,14 @@ const machine = produce(
             break;
           }
         }
+        draft.awaitingUpdate = false;
+        break;
+      case "CHECK_MACHINE_POWER_START":
+        draft.awaitingUpdate = true;
+        break;
+      case "CHECK_MACHINE_POWER_ERROR":
```

Comment:
`"CHECK_MACHINE_POWER_SUCCESS` case is missing

Reply:
Interesting, are you not getting an `UPDATE_MACHINE_NOTIFY` when the power check completes?

Reply:
Nope, no `UPDATE_MACHINE_NOTIFY`. Maybe this only happens if  `CHECK_MACHINE_POWER` returns the same power state that the machine is in? i.e. MAAS thinks the machine is on, you check the power and it is indeed on = no update.

Reply:
Ah OK, I think that shows that we can't show this spinner after all, thanks for double checking.